### PR TITLE
Add Xenus library

### PR DESCRIPTION
**Pull request by abellion**
Following https://github.com/mongodb/docs-ecosystem/pull/437.

### CODE REVIEW

File: source/php-libraries.txt [hunk 1]
```diff
@@ -37,6 +37,10 @@ Stand-alone Libraries
   **mongodb** extension. It is schema-less and supports fast object hydration
   and persistence, which makes it well-suited for modeling aggregation results.
 
+- `Xenus <https://github.com/abellion/xenus>`_ is an elegant MongoDB ODM
+   that supports events, relationships, embedded documents, and more.
+   It works on PHP 7+ and has an integration to the Laravel framework.
```
Comment (jmikola) [inline source/php-libraries.txt:42]: These two lines are indented an extra space, which might generate an RST warning. I think they should align with the backtick on the first line of the bullet.

File: source/php-libraries.txt [hunk 2]
```diff
@@ -60,7 +64,11 @@ Framework Integrations
     library and includes an abstract model, a helper class for handling bulk
     writes, database wrappers for Laravel and Lumen, and some necessary helper
     functions.
-    
+
+  - `Xenus Laravel <https://github.com/abellion/xenus-laravel>`_ integrates
+    Xenus to the Laravel and Lumen frameworks, adding failed jobs and migrations
```
Comment (jmikola) [inline source/php-libraries.txt:69]: I'd rewrite as "integrates Xenus ODM with the Laravel...". Aside from s/to/with/, referring to it as "Xenus ODM" hints at the library's purpose without expecting the reader to see the earlier bullet on the page.

**Comment by abellion** [comment]
Thanks for the review @jmikola.

**Comment by jmikola** [comment]
@ccho-mongodb: this is OK to merge IMO

**Pull request merged**